src/components/MarkdownProvider/components/ColorPalette.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { readableColor } from 'polished';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const StyledColorHex = styled.figcaption`
+  margin-left: auto;
+  font-size: ${p => p.theme.fontSizes.sm};
+`;
+
+const StyledColorPalette = styled.div`
+  margin-top: ${p => p.theme.space.md};
+  margin-bottom: ${p => p.theme.space.xl};
+`;
+
+const StyledColorSwatch = styled.figure<{ color: string }>`
+  display: flex;
+  align-items: center;
+  background-color: ${p => p.color};
+  padding: ${p => p.theme.space.xs} ${p => p.theme.space.sm};
```

Comment:
After seeing the whole page of palettes, I'd vote for this to move to `padding: ${p => p.theme.space.sm};`